test.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# Usage: ./test.sh <make target>
+# Execute a makefile target inside a Docker environment
+
+MAKE_TARGET="test-coverage-all-codeclimate"
+if [[ -n "$1" ]]; then
+  MAKE_TARGET="$1"
+fi
+
+docker stop commander-int-ssh-server || true
+docker rm commander-int-ssh-server || true
+docker stop commander-integration-go-test || true
+docker rm commander-integration-go-test || true
+docker network rm commander_test || true
+
+docker build -t commander-int-ssh-server -f integration/containers/ssh/Dockerfile .
+docker network create  --driver=bridge --subnet=172.28.0.0/16 commander_test
+
+docker run -d \
+  --ip=172.28.0.2 \
+  --network commander_test \
+  --name int-ssh-server \
+  --name commander-int-ssh-server \
+  commander-int-ssh-server
+
+docker build -t commander-int-test -f integration/containers/test/Dockerfile .
+docker run \
```

Comment:
you can always add `docker run --rm` here. Avoiding some of the code below. But, you probably have a valid reason for not doing so

Reply:
👍  Just forgot it :D
Added `--rm`, time shows how this container evolves. If it is used more often it definitely needs to change and cached.

Reply:
You can remove lines, 40 and 41 as well

Reply:
Right, done